stan/math/prim/prob/bernoulli_lcdf.hpp

```diff
@@ -69,12 +71,10 @@ return_type_t<T_prob> bernoulli_lcdf(const T_n& n, const T_prob& theta) {
       continue;
     }
 
-    const T_partials_return Pi = 1 - theta_vec.val(i);
-
-    P += log(Pi);
+    P += theta_vec.val(i);
 
     if (!is_constant_all<T_prob>::value) {
-      ops_partials.edge1_.partials_[i] -= inv(Pi);
+      ops_partials.edge1_.partials_[i] -= partials_vec.val(i);
```

Comment:
Can this be vectorized to this?

```
op_partials.edge1_ -= partials_vec;
```

Also, `partials_vec` should be renamed to `neg_partials_vec` because it's the negation.